React InstantSearch/routing-seo-friendly/src/App.js

```diff
@@ -133,22 +133,27 @@ Hit.propTypes = {
   hit: PropTypes.object.isRequired,
 };
 
+let timeout;
```

Comment:
We should use a ref here instead of a global variable: https://github.com/algolia/doc-code-samples/blob/8a3ffbec4b1661c5a997e612e47a075e1c63a3be/React%20InstantSearch/routing-basic/src/App.js#L35-L37